include/admin_navigation.js

```diff
@@ -379,46 +446,70 @@ module.exports = function AdminNavigationModule(pb) {
      * @param {String} parentId
      * @param {Object} node
      * @returns {Boolean}
+     * @param site
      */
-    AdminNavigation.addChild = function (parentId, node) {
-        if (isDuplicate(node.id)) {
+    AdminNavigation.addChildToSite = function (parentId, node, site) {
+        if (exists(node.id, site)) {
             return false;
         }
 
-        if (!AdminNavigation.childrenAdditions[parentId]) {
-            AdminNavigation.childrenAdditions[parentId] = [];
+        var additionsMap;
+        if (!(site in AdminNavigation.childrenAdditions)) {
+            additionsMap = AdminNavigation.childrenAdditions[site] = {}
+        } else {
+            additionsMap = AdminNavigation.childrenAdditions[site];
         }
 
-        AdminNavigation.childrenAdditions[parentId].push(node);
+        if (!additionsMap[parentId]) {
+            additionsMap[parentId] = [];
+        }
+
+        additionsMap[parentId].push(node);
         return true;
     };
 
+    AdminNavigation.addChild = function (parentId, node) {
+        return AdminNavigation.addChildToSite(parentId, node, GLOBAL_SITE);
+    };
+
     /**
      * Adds a new top level node
      * @static
-     * @method add
+     * @method addToSite
      * @param {Object} node
      * @returns {Boolean}
+     * @param site
      */
-    AdminNavigation.add = function (node) {
-        if (isDuplicate(node.id)) {
+    AdminNavigation.addToSite = function (node, site) {
+        if (exists(node.id, site)) {
             return false;
         }
 
-        AdminNavigation.additions.push(node);
+        if (!(site in AdminNavigation.additions)) {
+            AdminNavigation.additions[site] = [];
+        }
+        AdminNavigation.additions[site].push(node);
         return true;
     };
 
+    AdminNavigation.add = function (node) {
+        return AdminNavigation.addToSite(node, GLOBAL_SITE);
+    };
+
+    AdminNavigation.remove = function (id) {
+        return AdminNavigation.removeFromSite(id, GLOBAL_SITE);
+    };
```

Comment:
There is no need for the ```add``` and ```remove``` functions.  The ```removeFromSite``` and ```addToSite``` should be modified to default the second parameter rather than wrapping in a default function.  It is a extra code to test and a performance degradation due to the extra function call.